src/OrchardCore.Modules/OrchardCore.Media/Services/MediaCacheBackgroundTask.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+using OrchardCore.BackgroundTasks;
+
+namespace OrchardCore.Media.Services
+{
+
+    // At 12:00 on Monday.
+    [BackgroundTask(Schedule = "0 12 * * 1", Description = "Performs cleanup operations for ms-cache and is-cache folders At 12:00 on Monday.")]
```

Comment:
It would be nice to make the period configurable

Reply:
And also make it a feature so that it can be disabled.

Reply:
It also makes my life easier while debugging the MediaFileStoreResolverMiddleware and DefaultMediaFileStoreCacheFileProvider from the other PR.

Reply:
@hishamco 

> It would be nice to make the period configurable

It is already overridable by enabling the BackgroundTask feature that allows to override the settings of any bg task through the admin UI

Reply:
Thanks @jtkech I didn't know that because I didn't work with background tasks in OC ;)

Reply:
![image](https://user-images.githubusercontent.com/3228637/110010260-17e5e300-7cec-11eb-9fbb-4d205457460f.png)
![image](https://user-images.githubusercontent.com/3228637/110010276-1c120080-7cec-11eb-97b2-1af1ab872242.png)

Will change the description to not include the CRON period.

Reply:
French ;) but I got the point ... thanks

Reply:
Updated them. To be honest I did not even knew about this one either 😄 🙈

Reply:
No need .. I got the idea, thanks Jasmin